frontends/web/src/locales/en/app.json

```diff
@@ -1030,27 +1030,27 @@
       "title": "Passphrase enabled"
     },
     "successEnabled": {
-      "message": "Optional passphrase <strong>successfully disabled</strong>!\nYou will not be asked to provide a passphrase anymore.\nPlease replug your BitBox02 now.",
+      "message": "Optional passphrase <strong>successfully disabled</strong>!\n\nYou will not be asked to provide a passphrase anymore. Please replug your BitBox02 now.",
       "title": "Passphrase disabled"
     },
     "summary": {
       "button": "Enable passphrase",
       "title": "Summary",
-      "understand": "I understand how the passphrase works and the risks associated with it.",
+      "understand": "",
       "understandList": [
         "The passphrase is an additional layer of security on top of your backup.",
         "Entering a different passphrase will always generate a different wallet.",
         "To restore your wallet you need <strong>both the passphrase and backup</strong>."
       ]
     },
     "what": {
-      "button": "How does this work?",
-      "message": "A wallet is created (derived) from a very big random number, also known as seed or wallet-seed. It is created when you first set up your BitBox02 and is backed up with the microSD cards or 24 words. Anyone who has access to the seed has full control over the funds on that wallet, therefore the seed is a secret.\n\nA passphrase is an optional secret, added to the seed, to create a completely different wallet based on seed + passphrase. A passphrase can be anything: letters, words, special characters.\n\nTo restore your passphrase enabled wallet you will need the passphrase + wallet backup. If you forget your passphrase, you will not be able to access your wallet, even if you have your backup!",
-      "title": "What is a Passphrase?"
+      "button": "Learn how this works",
+      "message": "A wallet is created (derived) from a very big random number, also known as a seed. This seed is created when you first set up your BitBox02 and is backed up with the microSD card or 24 words. Anyone who has access to the seed has full control over the funds on that wallet.\n\nA passphrase is an <strong>optional secret</strong>, added to the seed. When using a passphrase, each passphrase creates a new wallet based on the seed + passphrase (optional secret). A passphrase can be anything: letters, words, special characters or it can even be empty. The default wallet is in fact derived from the seed + empty passphrase.",
+      "title": "What is a passphrase?"
     },
     "why": {
-      "button": "Why use a passphrase?",
-      "message": "The BitBox02 protects the seed against extraction, but the backup (microSD card or 24 words) gives full access to your wallet. That is why it should be stored in a secure location!\n\nWhen using a passphrase, each passphrase unlocks a new wallet. A different passphrase will create a different wallet.",
+      "button": "Why use a passphrase",
```

Comment:
why no question mark?

Reply:
yes, this is the button text, the actual screen title has a question mark